modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -260,6 +264,34 @@ protected function getTileSourceFromRow(ResultRow $row, $iiif_address, $iiif_bas
     return $canvases;
   }
 
+  /**
+   * Pull a title from the node or media passed to this view.
+   *
+   * @param string $content_path
+   *   The path of the content being requested.
+   *
+   * @return string
+   *   The entity's title.
+   */
+  public function getEntityTitle(string $content_path): string {
+    $entity_title = $this->t('IIIF Manifest');
+    try {
+      $params = Url::fromUserInput($content_path)->getRouteParameters();
+      if (isset($params['node'])) {
+        $node = \Drupal::entityTypeManager()->getStorage('node')->load($params['node']);
+        $entity_title = $node->getTitle();
+      }
+      elseif (isset($params['media'])) {
+        $media = \Drupal::entityTypeManager()->getStorage('media')->load($params['media']);
```

Comment:
Should... the `entity_type.manager` service be being injected instead of used from the global container dealio?

... Like, would expect PHPCS to be complaining about this, but... I guess I typically run it with more things being checked... [looks like `islandora_ci` only hits the `Drupal`](https://github.com/Islandora/islandora_ci/blob/831d8f29f790dda71d4a071425782f710b9ca09c/travis_scripts.sh#L16) `--standard`, but... might be a good idea to add `DrupalPractice` as well... though I would assume that would go boom elsewhere, so... probably not worth going that far for now... _ramble_ https://github.com/Islandora/documentation/issues/1620 is talking about introducing it.... though yeah...

... probably fine to leave as-is for now, if we want.

Reply:
Good point. Changed to use dependency injection.